app/gui2/src/components/ComponentBrowser.vue

```diff
@@ -43,7 +43,7 @@ const props = defineProps<{
 
 const emit = defineEmits<{
   accepted: [searcherExpression: string, requiredImports: RequiredImport[]]
-  closed: [searcherExpression: string, requiredImports: RequiredImport[]]
+  closed: [searcherExpression: string, requiredImports: RequiredImport[], anyInputChange: boolean]
```

Comment:
Do we need a `closed` event? I think the CB could just emit `accepted` or `canceled`.

Reply:
"Closed" means that it lost focus (due to background click, for example). Since this PR, it does differ from direct accept.

Reply:
Yes. I don't see the need to increase the complexity of the ComponentBrowser/GraphEditor interface by exposing this distinction to the GraphEditor. The CB could decide whether to emit `accepted` or `canceled` internally, and keep `anyInputChange` encapsulated.